SCAutolib/cli_commands.py

```diff
@@ -165,7 +169,47 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
+@click.command()
+@click.option('--init',
+              required=False,
+              default=False,
+              is_flag=True,
+              help="Initialize GUI for manual testing")
+@click.option("--assert-text",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Check if a word is found on the screen ")
+@click.option("--click-on",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Click on object containing word")
+@click.option("--check-home-screen",
```

Comment:
Probably we will also need a --check-no-home-screen for negative tests or something.